exercises/ruby-1-b/.meta/config.json

```
@@ -3,39 +3,72 @@
   "test_file": "ruby_1_b_test.rb",
   "tests": [
     {
-      "test": "test_one_word_with_one_vowel",
-      "name": "one word with one vowel",
-      "cmd": "word_with_most_vowels(\"at\")"
+      "name": "test_find_by_empty_isbn_returns_nil",
+      "cmd": "@library.find(\"\")"
```

In the `ruby-1-a` example, we're giving the full code used to test the input. See https://github.com/exercism/research_experiment_1/blob/master/exercises/ruby-1-a/.meta/config.json#L14.

Apply that to this test, this would become:

```suggestion
      "cmd": "library = Library.new\nlibrary.find(\"\")"
```

Could you update the other entries too?